start/src/main/java/org/apache/accumulo/start/Main.java

```diff
@@ -41,63 +41,57 @@
-      } catch (ClassNotFoundException e) {
-        log.error("Unable to find Hadoop Configuration class on classpath, check configuration.",
-            e);
-        System.exit(1);
-      }
-      Object conf = null;
-      try {
-        conf = confClass.getDeclaredConstructor().newInstance();
-      } catch (Exception e) {
-        log.error("Error creating new instance of Hadoop Configuration", e);
-        System.exit(1);
-      }
-      try {
-        Method getClassByNameOrNullMethod =
-            conf.getClass().getMethod("getClassByNameOrNull", String.class);
-        getClassByNameOrNullMethod.invoke(conf, "org.apache.hadoop.mapred.JobConf");
-        getClassByNameOrNullMethod.invoke(conf, "org.apache.hadoop.mapred.JobConfigurable");
-      } catch (Exception e) {
-        log.error("Error pre-loading JobConf and JobConfigurable classes, VFS classloader with "
-            + "system classes in HDFS may not work correctly", e);
-        System.exit(1);
-      }
-
-      if (args.length == 0) {
-        printUsage();
-        System.exit(1);
-      }
-      if (args[0].equals("-h") || args[0].equals("-help") || args[0].equals("--help")) {
-        printUsage();
-        System.exit(1);
-      }
-
-      // determine whether a keyword was used or a class name, and execute it with the remaining
-      // args
-      String keywordOrClassName = args[0];
-      KeywordExecutable keywordExec = getExecutables(loader).get(keywordOrClassName);
-      if (keywordExec != null) {
-        execKeyword(keywordExec, stripArgs(args, 1));
-      } else {
-        execMainClassName(keywordOrClassName, stripArgs(args, 1));
-      }
+      @SuppressWarnings("deprecation")
+      var deprecatedConfClass = org.apache.accumulo.start.classloader.AccumuloClassLoader
+          .getClassLoader().loadClass("org.apache.hadoop.conf.Configuration");
+      confClass = deprecatedConfClass;
+    } catch (ClassNotFoundException e) {
+      log.error("Unable to find Hadoop Configuration class on classpath, check configuration.", e);
+      throw e;
+    }
+    Object conf = null;
+    try {
+      conf = confClass.getDeclaredConstructor().newInstance();
+    } catch (Exception e) {
+      log.error("Error creating new instance of Hadoop Configuration", e);
+      throw e;
+    }
+    try {
+      Method getClassByNameOrNullMethod =
+          conf.getClass().getMethod("getClassByNameOrNull", String.class);
+      getClassByNameOrNullMethod.invoke(conf, "org.apache.hadoop.mapred.JobConf");
+      getClassByNameOrNullMethod.invoke(conf, "org.apache.hadoop.mapred.JobConfigurable");
+    } catch (Exception e) {
+      log.error("Error pre-loading JobConf and JobConfigurable classes, VFS classloader with "
+          + "system classes in HDFS may not work correctly", e);
+      throw e;
+    }
 
-    } catch (Throwable t) {
-      log.error("Uncaught exception", t);
+    if (args.length == 0) {
+      printUsage();
       System.exit(1);
```

Comment:
Even though this is a legitimate use of exit, if we want consistency with the help options, we could just return without an exit code here as well. I'm fine either way, as long as we don't throw an exception here.

Reply:
@dlmarion Not sure if you saw this comment. I made it after marking "Approve" on my last review. Feel free to mark this as "resolved" if you're fine with it as-is.

Reply:
I'm fine with it as is.